canonicalwebteam/blog/django/views.py

```diff
@@ -51,13 +51,13 @@ def article_redirect(request, slug, year=None, month=None, day=None):
 
 def article(request, slug):
     try:
-        article = api.get_article(slug)
+        article = api.get_article(slug, tags_id)
```

Comment:
Shouldn't this variable be `tag_ids`?

Reply:
This presumably also means it should be `tags_id = settings.BLOG_CONFIG["TAGS_ID"]`, which changes the external interface and so also implies changing the settings in cn.ubuntu.com. But this should be acceptable, it just requires at least a minor version change.